src/20_Components/amp-bind.html

```diff
@@ -118,17 +121,30 @@
     <p [class]="allAnimals[currentAnimal].style" class="greenBackground">Each animal has a different background color.</p>
     <!-- ## Binding Attribute Values -->
     <!--
-      Various AMP components support binding attribute values with `amp-bind`. For example, `amp-img` URLs can be changed.
+      Various AMP components support binding attribute values with `amp-bind` and here it follows
```

Comment:
One minor grammatical change:

"Various AMP components support binding attribute values with `amp-bind`. Below you can find a subset of examples"

(Note that as the list grows, we may want to have a simple list of links at the top to quickly jump to each sample)